benchbuild/project.py

```diff
@@ -322,16 +322,26 @@ def source_of(self, name: str) -> tp.Optional[str]:
         """
         Retrieve source for given index name.
 
+        First we try to find the given source in the configured variant index.
+        If unsuccessful, we try to find them in our configured sources. The
+        second variant picks up all sources that did not take part in
+        version expansion.
```

Comment:
Maybe a bit out of context, but that means that we probably might want to have a shared git cache when using containers to avoid repeated clones when accessing such additional sources, doesn't it?

Reply:
Good point, I need to try it first